Sources/MySQL/Database/MySQLDatabaseConfig.swift

```diff
@@ -20,12 +20,20 @@ public struct MySQLDatabaseConfig {
     /// Database name.
     public let database: String
 
+    /// Character set. Default utf8_general_ci
+    let characterSet: MySQLCharacterSet
+
     /// Creates a new `MySQLDatabaseConfig`.
-    public init(hostname: String = "127.0.0.1", port: Int = 3306, username: String, password: String? = nil, database: String) {
+    public init(hostname: String = "127.0.0.1", port: Int = 3306, username: String, password: String? = nil, database: String, characterSet: String = "utf8_general_ci") throws {
```

Comment:
I think it would be better to expose `MySQLCharacterSet` instead of string here. If you want to  use a String, use the `init(string:)` method on `MySQLCharacterSet`. That allows us to make this init non-throwing and let's users use a pre-defined case or even an integer which is more powerful.

Reply:
Done